test/Generator/MethodGeneratorTest.php

```diff
@@ -375,4 +375,18 @@ public function testFromByReferenceMethodReflection()
 
         self::assertStringMatchesFormat('%Apublic function & byRefReturn()%A', $methodGenerator->generate());
     }
+
+
+    public function testClearBodyIndentation()
+    {
+        $class = new \ReflectionClass(MethodGenerator::class);
+        $method = $class->getMethod('clearBodyIndention');
+        $method->setAccessible(true);
+
+
+        $testMethod = '/** @return bool */ public function someMethod() { return false; }';
+        $clearedMethod = $method->invokeArgs( null, ['\t\t' . $testMethod]);
```

Comment:
This test uses `protected` API calls to trigger the failure. I can't do much with this - can you write one that uses the `public` API?